From 238f753f0e7ffae2a73356377ba4178b2d12a193 Mon Sep 17 00:00:00 2001
From: Peter Jones <pjones@redhat.com>
Date: Tue, 8 Aug 2017 13:08:59 -0400
Subject: [PATCH 097/225] docs/Makefile.am: use info-in-builddir

This makes it so builds in subdirectories don't contaminate the main
build directory with their .info files.  While the result is two
identical targets built, it still makes packaging more straightforward.

Signed-off-by: Peter Jones <pjones@redhat.com>
---
 docs/Makefile.am | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/docs/Makefile.am b/docs/Makefile.am
index 93eb39627..e396ba10f 100644
--- a/docs/Makefile.am
+++ b/docs/Makefile.am
@@ -1,4 +1,4 @@
-AUTOMAKE_OPTIONS = subdir-objects
+AUTOMAKE_OPTIONS = subdir-objects info-in-builddir
 
 # AM_MAKEINFOFLAGS = --no-split --no-validate
 info_TEXINFOS = grub.texi grub-dev.texi
-- 
2.14.3

